pkgs/by-name/ni/nixos-rebuild-ng/package.nix

```diff
@@ -1,14 +1,19 @@
 {
   lib,
+  stdenv,
   installShellFiles,
   mkShell,
   nix,
-  nixos-rebuild,
   python3,
   python3Packages,
   runCommand,
+  scdoc,
   withNgSuffix ? true,
+  withShellFiles ? (stdenv.buildPlatform.canExecute stdenv.hostPlatform),
```

Comment:
*Technically*, we can install the manpage despite the conditional here.

Reply:
Actually, you may be right and things may just work here. And missing the man page would be bad since the man page is used in place of help.

Let me try to compile `pkgCross.<arch>.nixos-rebuild-ng` with this value set to true to see what happens.

Reply:
Changed this to `true`. I think it should work fine for both the man page and the shell completion (that will run in native Python but this shouldn't matter).

If this is an issue we can always fix later.